faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -495,7 +538,10 @@ public CompletableFuture<HttpResponse<Flow.Publisher<List<ByteBuffer>>>> perform
     final CompletableFuture<HttpResponse<Flow.Publisher<List<ByteBuffer>>>> rv = new CompletableFuture<>();
     HttpRequest request;
     try {
-      request = makeHttpRequest(httpMethod, path, Optional.of(body), params, Optional.empty(), HttpClient.Version.HTTP_2);
+      // TODO Implement traceId/tags for streaming requests
```

Comment:
is that something we want to do?

I'm not sure there's a use case for that.

Reply:
Seems like we'll need to now based on https://faunadb.atlassian.net/browse/FE-2748. Tags, I think, make sense - so we should probably enable that use case. TraceId _could_ make sense if grouping all of the queries into the same "trace forest" makes sense...which we would do by giving them the same trace identifier, e.g. `00-{someIdentifierSharedByAllStreamedQueries}-{perQueryIdentifier}-00`. For now, I've left the TODO, we can discuss separately in JIRAs to distill the requirement.

Reply:
Based on this quote from the ticket:

> DX won’t add support for the trace header or tags to the stream methods in the drivers since it won’t be piped through in the same way as transactional logs

I don't think we do :D

we just need to give them access to the logs.

Reply:
Haha, yeah, we don't. I think I made this comment before I saw Adam's response.